projects/canopy/src/styles/mixins.scss

```diff
@@ -144,7 +144,7 @@ $breakpoints: (
   background-color: var(--#{$variant}-bg-color) !important;
   color: var(--#{$variant}-color) !important;
 
-  a {
+  a:not(.lg-btn) {
```

Comment:
Note this is inside the variant CSS, so it becomes `.lg-variant--success a:not(.lg-btn)` for example

Reply:
I think thats ok, I assume we would never directly apply the `variant` directive to the link anyway.